cdc/api/v2/model.go

```diff
@@ -693,6 +706,14 @@ type ConsistentConfig struct {
 	UseFileBackend        bool   `json:"use_file_backend"`
 	Compression           string `json:"compression,omitempty"`
 	FlushConcurrency      int    `json:"flush_concurrency,omitempty"`
+
+	MemoryUsage *ConsistentMemoryUsage `json:"memory_usage"`
+}
+
+// ConsistentMemoryUsage represents memory usage of Consistent module.
+type ConsistentMemoryUsage struct {
+	MemoryQuotaPercentage uint64 `json:"memory_quota_percentage"`
```

Comment:
redo_memory_quota_percentage

Reply:
The struct is named `ConsistentMemoryUsage`, with the `Consistent` prefix. so internal fields can be shorter.